nestkernel/simulation_manager.cpp

```diff
@@ -375,6 +376,61 @@ nest::SimulationManager::simulate( Time const& t )
   t_slice_begin_ = timeval();
   t_slice_end_ = timeval();
```

Comment:
And it looks like value needs to be nulled in order to check whether it was set at all -- the tv_sec is checked to see if it's zero (not precisely correct) to see whether it's been set somewhere else.

This code is messy, probably should be fixed, but outside of the purview of the current PR.